lib/phoenix_html/tag.ex

```diff
@@ -87,16 +88,46 @@ defmodule Phoenix.HTML.Tag do
   def content_tag(name, content, attrs) when is_list(attrs) do
     name = to_string(name)
     {:safe, escaped} = html_escape(content)
-    {:safe, [?<, name, build_attrs(attrs), ?>, escaped, ?<, ?/, name, ?>]}
+
+    {:safe,
+     [?<, name, build_attrs(attrs) |> Enum.sort() |> tag_attrs(), ?>, escaped, ?<, ?/, name, ?>]}
+  end
+
+  @doc """
+  Escapes a list of attributes, returning iodata.
+
+      iex> attributes_escape(title: "the title", id: "the id", selected: true)
+      {:safe,
+       [
+         [32, "title", 61, 34, "the title", 34],
+         [32, "id", 61, 34, "the id", 34],
+         [32, "selected"]
+       ]}
+
```

Comment:
We should document that, opposite to `tag` and `content_tag`, the attributes are not sorted.